starter_code/views/page-movie.hbs

```diff
@@ -0,0 +1,13 @@
+
+
+
+
+<div class= "un">
+    <div class = "deux"><img src = "{{movie.image}}"/></div>
+    <div class = "trois"><div class = "quatre">{{movie.title}}</div> {{movie.director}}</div>
+    <div class = "cinq">{{movie.description}}</div>
+    <div class = "six"><ul>{{#each movie.star}}
```

Comment:
The key is movie.stars, movie.star is undefined.
You beautiful and ergonomic design wasn't showing the stars !